src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
Actually for this simple operation you can use **_for loop_** in order to create and print information about ball